src/reconcile/rec_write.c

```diff
@@ -26,6 +26,11 @@ typedef struct {
 	uint32_t flags;			/* Caller's configuration */
 
 	WT_ITEM	 disk_image;		/* Temporary disk-image buffer */
+	/*
+	 * Temporary buffer used to write out a disk image when managing two
+	 * chunks worth of data in memory
+	 */
+	WT_ITEM *interim_buf;
```

Comment:
yes, I see performance improvement when I create the buffer once and keep it around, instead of creating and destroying each time in __rec_split_write_prev_and_shift_cur()